<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCT_Simulator</class>
 <widget class="QMainWindow" name="MCT_Simulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MCT_Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>90</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="batteryText">
       <property name="text">
        <string>100%</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="styleSheet">
        <string notr="true">QListWidget{
rgb(239, 41, 41)
}</string>
       </property>
       <item>
        <property name="text">
         <string>Main Menu</string>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>239</red>
           <green>41</green>
           <blue>41</blue>
          </color>
         </brush>
        </property>
        <property name="foreground">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>52</red>
           <green>101</green>
           <blue>164</blue>
          </color>
         </brush>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Programs</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Frequency</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>History</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Recording Setting</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>rangeChanged(int,int)</signal>
   <receiver>batteryText</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
